src/Microsoft.Azure.WebJobs.Host/Bindings/DefaultAttributeInvokerDescriptor.cs

```diff
@@ -9,26 +9,16 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Bindings
 {
-    // Default implementation of IAttributeInvokeDescriptor which can be operate on any attribute and 
-    // follows basic conventions with a [ResolveProperty]. 
-    // Assume any property without a [ResolveProperty] attribute is read-only. 
-    internal class DefaultAttributeInvokerDescriptor<TAttribute> : IAttributeInvokeDescriptor<TAttribute>
+    // Helpers for providing default implementations of IAttributeInvokeDescriptor
+    internal static class DefaultAttributeInvokerDescriptor<TAttribute>
```

Comment:
Seems like this type should be renamed since it isn't an InvokerDescriptor anymore